### ISSUE TITLE
No filter predicate is being generated for filtered indexes

### ISSUE THREAD

**Issue by pyanguas** [sethreno/schemazen#67]
Script generated by SSMS for a filtered index (unique in this case):

```
CREATE UNIQUE NONCLUSTERED INDEX [IXUK_tbl_ventsClientes_1] ON [dbo].[tbl_ventsClientes]
(
[NIF] ASC, [TipoCliente] ASC, [IDEstado] ASC
)

WHERE ([NIF] IS NOT NULL)
```

Same script, generated by SchamaZen 1.26:

```
CREATE UNIQUE NONCLUSTERED INDEX [IXUK_tbl_ventsClientes_1] ON [dbo].[tbl_ventsClientes]
(
[NIF], [TipoCliente], [IDEstado]
)
```

Database: SQL Server 2008 R2

**Comment by benjamin-hodgson** [comment]
This issue is affecting me too.

**Comment by kdd21** [comment]
Just ran into this one as well.

**Comment by sethreno** [comment]
This was fixed in version 1.3.40
